tethysapp/hydro_var_monitor/templates/hydro_var_monitor/home.html

```diff
@@ -3,298 +3,177 @@
-
-          <p><span class="bolded">Comparar los promedios por una variable</span></p>
-        <p>Seleccione la región y variable según lo descrito anteriormente, y posteriormente podrá comparar promedios. Observará un gráfico con la información sobre
-            los promedios comparando los datos obtenidos de una variable con diferentes satélites.</p>
-          <p><span class="bolded">Selección de una fecha determinada</span></p>
-          <p> Podrá visualizar los resultados por un año específico. Haciendo clic en el botón <span class="bolded">“Usar una Fecha Específica”</span>.</p>
-      </div>
-      <div class="modal-footer">
-        <button type="button" class="btn btn-outline-secondary" data-bs-dismiss="modal">Cerrar</button>
-      </div>
+        </div>
     </div>
+    <div id="errorModal" class="modal fade" role="dialog">
+        <div class="modal-dialog">
+
+            <!-- Modal content-->
+            <div class="modal-content">
+                <div class="modal-header">
+                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                    <h4 class="modal-title">Error</h4>
+                </div>
+                <div class="modal-body">
+                    <p>Por favor elimine la entrada anterior.</p>
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-outline-secondary" data-bs-dismiss="modal">Cerrar</button>
+                </div>
+            </div>
 
-  </div>
-</div>
-<div id="errorModal" class="modal fade" role="dialog">
-  <div class="modal-dialog">
-
-    <!-- Modal content-->
-    <div class="modal-content">
-      <div class="modal-header">
-        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
-        <h4 class="modal-title">Error</h4>
-      </div>
-      <div class="modal-body">
-        <p>Por favor elimine la entrada anterior.</p>
-      </div>
-      <div class="modal-footer">
-        <button type="button" class="btn btn-outline-secondary" data-bs-dismiss="modal">Cerrar</button>
-      </div>
+        </div>
     </div>
-
-  </div>
-</div>
-<div id="lat-lon-modal" class="modal fade" role="dialog">
-  <div class="modal-dialog">
-
-    <!-- Modal content-->
-    <div class="modal-content">
-        <div class="modal-header">
-          <h5 class="modal-title" id="exampleModalLabel">Poner Latitud y Longitud </h5>
-          <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+    <div id="lat-lon-modal" class="modal fade" role="dialog">
+        <div class="modal-dialog">
+
+            <!-- Modal content-->
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title" id="exampleModalLabel">Poner Latitud y Longitud </h5>
```

Comment:
needs to be in english or needs to be in a file which is different for each language